airbyte-integrations/connectors/source-shopify/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 9da77001-af33-4bcd-be46-6252bf9342b9
-  dockerImageTag: 0.4.0
+  dockerImageTag: 0.5.0
```

Comment:
If we only extend  the `orders` stream schema with additional fields, as well as `transactions` stream, the change would be `non-breaking` and the version in this case would be: `0.4.1`